experimental/Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: all vet lint build test clean
+
+dirs=$(shell find . -maxdepth 2 -name *.go -printf "%h\n" | uniq)
+targets=$(join $(patsubst %,%/,$(dirs)),$(dirs))
+
+all: $(targets)
+
+define T
+$1/$1: $1/*.go
+	go get -tags netgo ./$$(@D)
+	go build -ldflags "-extldflags \"-static\"" -tags netgo -o $$@ ./$$(@D)
+endef
+
+$(foreach d,$(dirs),$(eval $(call T,$d)))
```

Comment:
I think this would be nicer if we just had individual, dumb, copy-pasted targets, removing dirs/targets/T/foreach